finary_uapi/currencies.py

```diff
@@ -30,3 +30,12 @@ def get_cryptocurrency_by_code(session: requests.Session, code: str):
                 return currency
         return currency
     return {}
+
+def get_fiatcurrency_by_code(session: requests.Session, code: str):
```

Comment:
It can be useful thanks! it's very similar to the previous function though, so you should create a new function get currency_by_code and call it from the fiat and crypto ones.

Reply:
Something like this and potentially depreciating afterward get_cryptocurrencies and get_fiatcurrency functions?

```python
def get_currency_by_code(session: requests.Session, type: str, code: str):
     """
    type = 'crypto' or 'fiat'
    """
    currencies = get_currencies(session, type, code)
     if len(currencies["result"]) > 0:
        currency = currencies["result"][0]
        for currency in currencies["result"]:
            if currency["code"] == code:
                return currency
        return currency
    return currency
```

Reply:
Yes